<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рабочий профиль</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="activeList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Активные</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Завершенные</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addNewJob">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/note_add.png</normaloff>icons/note_add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stats">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/statistics.png</normaloff>icons/statistics.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="monthList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Январь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Февраль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Март</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Апрель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Май</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Июнь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Июль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Август</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сентябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Октябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ноябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Декабрь</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="yearList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Профиль:</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>{username}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>30</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>В текущем месяце заработок составляет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="profit">
    <property name="geometry">
     <rect>
      <x>826</x>
      <y>29</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>{profit}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>921</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Наименование</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Клиент</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Открыта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Стоимость</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прибыль</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Закрыта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Статус</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="changeUser">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сменить пользователя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="servicesForClient">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/account_box.png</normaloff>icons/account_box.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Администрирование</string>
    </property>
    <addaction name="adm_clients"/>
    <addaction name="adm_categories"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="adm_clients">
   <property name="text">
    <string>Клиенты</string>
   </property>
  </action>
  <action name="adm_categories">
   <property name="text">
    <string>Категории</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
